infrastructure/bicep/virtual-machines/load-balancer/main.bicep

```diff
@@ -0,0 +1,179 @@
+// --- PARAMETERS --- //
+@description('Azure Location/Region')
+param location string = resourceGroup().location
+
+@description('Name of the company')
+param compName string
+
+@description('Deployment environment')
+param env string
+
+@description('Name of the resource group')
+param rg string
+
+@description('Private IP address of the subnet')
+param privateIPAddress string
+
+//@description('Frontend- and Backend port')
+param frontendPort int = -1
+param backendPort int = -1
+
+@description('Name and port of the probe')
+param probeName string = ''
+param probePort int = -1
+
+// --- VARIABLES --- //
+var vm = 'vm'
+
+var virtualNetworkName = 'vnet-${compName}-${rg}-${env}'
+var subnetName = 'snet-${compName}-${rg}-${vm}-${env}'
+
+var loadBalancerName = 'lbi-${compName}-${rg}-${env}'
+var frontendIPConfigurationName = 'feic-${compName}-${vm}-${env}'
+var backendAddressPoolName = 'bep-${compName}-${vm}-${env}'

```

Comment:
frontendIPConfiguration belongs to the load balancer, so it should be named after it (`feic-${compName}-${rg}-${env}`)?